package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "poll",
+  "version": "1.0.0",
+  "description": "simple polling system",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/helinsv/poll.git"
```

Comment:
you can use git@github.com:helinsv/poll.git instead https if you will add your ssh keys to GitHub.